src/components/Pages/Search/Search.css

```diff
@@ -0,0 +1,31 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.ThemeProvider {
+  display: flex;
+  margin: 15px 0;
+  margin-left: 15px;
+  justify-content: space-evenly;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
I think this class has a misleading name, isn't this the wrapper for a search?

Reply:
I removed the complete Theme Provider CSS the styling run with the Search Field.